NEWS.md

```diff
@@ -1,12 +1,14 @@
 ## 20.2.0 IN-PROGRESS
 
+* Adds `effective location` attribute to holdings record (MODINVSTOR-669)
 * Defaults instance.previouslyHeld to false (MODINVSTOR-454)
-* Provides `instance-storage 7.7`
+* Provides `holdings-storage 4.6` (MODINVSTOR-669)
```

Comment:
Is this the correct merge? I thought `instance-storage` was changed after the last release?